webextensions/manifest/theme.json

```diff
@@ -45,6 +45,11 @@
           },
           "accentcolor": {
             "__compat": {
+              "status": {
```

Comment:
These `status` fields should actually be below the `support` block (yes, I know it's not alphabetical sorting).  Can you swap the location of the `status` block to go right after the `support` one as to maintain consistency with the rest of the files in BCD?